contracts/DappDevMint.sol

```diff
@@ -88,94 +58,40 @@ contract DappDevMint
   function mint(address _to, uint256 _amount)
     public
   {
-      require(isOwnerOrDelegate(msg.sender));
+      require(canMint[msg.sender]);
       assert(token.mint(_to, _amount));
   }
 
-  function voteNewOwner(address newOwner, bool yayVote)
-    public
-  {
-    require(isOwner(msg.sender));
-    // Sender hasn't already proposed this
-    require(!newOwnerProposals[newOwner].proposed[msg.sender]);
-    // Add votes for this proposal
-    newOwnerProposals[newOwner].proposed[msg.sender] = true;
-    if (yayVote)
-    {
-      newOwnerProposals[newOwner].yayVotes += 1;
-      // If majority votes to accept this, add to delegates and remove this
-      if (newOwnerProposals[newOwner].yayVotes > owners.length/2)
-      {
-        owners.push(newOwner);
-        delete newOwnerProposals[newOwner];
-      }
-    }
-    else
-    {
-      newOwnerProposals[newOwner].nayVotes += 1;
-      // If majority votes to reject this, remove from proposals
-      if (newOwnerProposals[newOwner].nayVotes > owners.length/2)
-      {
-        delete newOwnerProposals[newOwner];
-      }
-    }
-  }
+  /**
+   * @dev Function Change ownership on the faucet token
+   * @param newOwner The address that will become the new owner
+   */
+   function transferTokenOwnership(address newOwner) public {
+       require(isOwner(msg.sender));
+       token.transferOwnership(newOwner);
+   }
 
-  function voteNewDelegate(address newDelegate, bool yayVote)
-    public
-  {
-    require(isOwner(msg.sender));
-    // Sender hasn't already proposed this
-    require(!newDelegateProposals[newDelegate].proposed[msg.sender]);
-    // Add votes for this proposal
-    newDelegateProposals[newDelegate].proposed[msg.sender] = true;
-    if (yayVote)
-    {
-      newDelegateProposals[newDelegate].yayVotes += 1;
-      // If majority votes to accept this, add to delegates and remove this
-      if (newDelegateProposals[newDelegate].yayVotes > owners.length/2)
-      {
-        delegates.push(newDelegate);
-        delete newDelegateProposals[newDelegate];
-      }
-    }
-    else
-    {
-      newDelegateProposals[newDelegate].nayVotes += 1;
-      // If majority votes to reject this, remove from proposals
-      if (newDelegateProposals[newDelegate].nayVotes > owners.length/2)
-      {
-        delete newDelegateProposals[newDelegate];
-      }
-    }
+
+  /**
+   * @dev Function add minters/delegates
+   * @param newDelegate The address that will be allowed to mint tokens.
+   */
+   function addDelegate(address newDelegate) public {
+      //only owner can add delegate
+      require(isOwner(msg.sender));
+      canMint[newDelegate] = true;
+      emit AccessControl(newDelegate, true);
```

Comment:
This is a new feature for v0.4.21, make sure the pragma reflects this

Reply:
“‘emit’ keyword in Solidity” @AniketEngg https://medium.com/@aniketengg/emit-keyword-in-solidity-242a679b0e1a